includes/namespace.php

```diff
@@ -38,17 +38,18 @@
  * @return void
  */
 function setup_new_site() {
-	$story = new Story_Post_Type( new Experiments(), new Meta_Boxes() );
-	$story->init();
+	$story = new Story_Post_Type( new Experiments(), new Meta_Boxes(), new Decoder() );
+	$story->register();
 	$story->add_caps_to_roles();
 	if ( ! defined( '\WPCOM_IS_VIP_ENV' ) || false === \WPCOM_IS_VIP_ENV ) {
 		flush_rewrite_rules( false ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.flush_rewrite_rules_flush_rewrite_rules
 	}
 
 	$database_upgrader = new Database_Upgrader();
-	$database_upgrader->init();
+	$database_upgrader->register();
```

Comment:
Could be (for a later PR, as discussed):
```php
Services::get( 'database_upgrader' )->register()
```

Reply:
Let's handle this https://github.com/google/web-stories-wp/issues/5668